libs/application/templates/children-residence-change/src/fields/Overview/index.tsx

```diff
@@ -140,6 +158,9 @@ const Overview = ({ application, setBeforeSubmitCallback }: FieldBaseProps) => {
 
   const controlCode =
     requestFileSignatureData?.requestFileSignature?.controlCode
+  // TODO: Look into if we want to do this in a different way - using the application state seems wrong
+  const contactInfoKey = application.state === 'draft' ? 'parentA' : 'parentB'
```

Comment:
Totally agree.
We could compare the id of the application (which is parents a ssn) to the ssn of the logged in user. 
It would of course be ideal that we know which role the user is acting in, but don't think we have access to that :/